controller/environment.go

```diff
@@ -52,6 +54,10 @@ func ConvertEnvironments(envs []*environment.Environment) *app.EnvironmentsList
 }
 
 func (c *EnvironmentController) Create(ctx *app.CreateEnvironmentContext) error {
+	if !c.developerModeEnabled {
+		return app.JSONErrorResponse(ctx, errors.NewInternalErrorFromString("operation not supporated"))
```

Comment:
Return ctx.MethodNotAllowed() instead:
```
if !c.config.DeveloperModeEnabled() {
    return ctx.MethodNotAllowed()
}
```
You will need to add `d.MethodNotAllowed` to design package for that.

Reply:
done commit [a1a7b87](https://github.com/fabric8-services/fabric8-env/pull/2/commits/a1a7b877ef7fbfc5f8cebceac836a4fd086700dc)